reinforcement_learning/common/sagemaker_rl/ray_launcher.py

```diff
@@ -123,7 +123,7 @@ def get_all_host_names(self):
 
     def ray_init_config(self):
         num_workers = max(self.num_cpus, 3)
-        config = {"num_cpus": num_workers, "num_gpus": self.num_gpus}
+        config = {"num_cpus": num_workers, "num_gpus": self.num_gpus, "webui_host": '127.0.0.1'}
```

Comment:
This is the host to bind the dashboard server to. SageMaker doesn't have expose ssh of the provisioned ec2 instance. Should it be removed?

Reply:
This is a quick fix to avoid the dashboard error trace. Ray dashboard can not be used but this can resolve the error trace.

Reply:
By default (`webui_host=None`)  the dashboard listens to `127.0.0.1` (https://github.com/ray-project/ray/blob/releases/0.8.5/python/ray/serve/constants.py#L17), why would adding this avoid the error trace? Maybe try disabling `include-webui`.

Reply:
according to this: https://github.com/ray-project/ray/issues/7084

Reply:
I see.Since the port is still closed, how would the log look like after this change?

Reply:
The log will not have the binding error anymore. I think this is a bug on their side, nothing to do with close/open of the port

Reply:
But seems they fixed that in the latest wheel. And deprecate this param in the next version. But for now, we'll have to use this to get around the error